changelogs/current.yaml

```diff
@@ -434,6 +434,13 @@ new_features:
   change: |
     Added :ref:`routeName() <config_http_filters_lua_stream_info_route_name>` API to the Stream Info Object to get the
     name of the route matched by the filter chain.
+- area: oauth2
+  change: |
+    Add the option to specify samesite cookie attribute value for oauth2 supported cookies.
+    To specify samesite attribute, choose one of the value from ``strict``,``lax`` or ``none``. if not specified,
+    a default value of ``disabled`` will be assigned and there will be no samesite value in cookie. see
```

Comment:
nit: s/in cookie/in the cookie and s/see/See

Reply:
in the cookie attribute